app/views/questionnaire.py

```diff
@@ -171,12 +174,12 @@ def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-ar
                                                    survey_title=TemplateRenderer.safe_content(schema['title']))
         return thank_you_template
     else:
-        return _redirect_to_latest_location(collection_id, eq_id, form_type)
+        routing_path = path_finder.get_full_routing_path()
```

Comment:
why not use the function decorator here?

Reply:
@collisdigital could do but the fully calculated routing path is only required if the survey is not complete (for the redirect), not for rendering the thank you template.

Reply:
ok